docs/integrations/saas-cloud/druva.md

```diff
@@ -0,0 +1,117 @@
+| where severity matches "{{severity}}"
+| where client_os matches "{{client_os}}"
+| where event_type matches "{{event_type}}"
+| where event_state matches "{{event_state}}"
+| if(isNull(user_id),"-",user_id) as user_id
+| if(isNull(user_email),"-",user_email) as user_email
+| if(isNull(client_os),"-",client_os) as client_os
+| if(isNull(user_name),"-",user_name) as user_name
+| if(isNull(device_id),"-",device_id) as device_id
+| if(isNull(initiator),"-",initiator) as initiator
+| if(isEmpty(ip),"-",ip) as ip
+| count_distinct(user_id)
+```
+
+## Set up collection
+
+To set up [Cloud to Cloud Integration Druva Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source/) for the Druva App, follow the instructions provided. These instructions will guide you through the process of creating a source using the Druva Source category, which you will need to use when installing the app. By following these steps, you can ensure that your Druva App is properly integrated and configured to collect and analyze your Druva data.
+
+## Installing the Druva App​
+
+This section has instructions for installing the Druva App for Sumo Logic and descriptions of each of the dashboards.
+
+Locate and install the app you need from the **App Catalog**. If you want to see a preview of the dashboards included with the app before installing, click **Preview Dashboards**.
+1. From the **App Catalog**, search for the app and select it.
+1. Select **Add Integration** button to install the app.
+1. Configure **Druva App** using the steps described in the [Citrix Cloud-to-Cloud Source](/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/druva-source/). If you already have set up your data, skip this step by clicking on **Next**.
+1. Complete the following fields:
+   1. **Data Source**. Select either of these options for the data source:
+      * Choose **Source Category** and then choose a source category from the list.
+      * Select **Enter a Custom Data Filter** and type in a custom source category that starts with an underscore. For Example, `_sourceCategory=MyCategory`.
+    2. **Folder Name**. You can retain the existing name, or enter a name of your choice for the app. 
+    3. Select the **Location in Library** (the default is the **Personal** folder in the library), or click **New Folder** to add a new folder.
+1. Click **Next**.
+
+Once an app is installed, it will appear in your **Personal** folder, or other folder that you specified. You can share it with your organization.
+
+The panels will begin to fill automatically. It's worth noting that each panel gradually fills with data that matches the time range query and has been received since the panel was created. The results will not be available right away, but with some patience, you will be able to view full graphs and maps.
+
+## Viewing Druva dashboards
```

Comment:
Capitalize the first word and lowercase the rest. Here "Druva" is a proper noun so it will be in caps. Refer to our style guide and for more details, I'd suggest reading the MSTP guide as well. It will help you a lot.